ddtrace/span.py

```diff
@@ -57,6 +58,7 @@ def __init__(
         parent_id=None,
         start=None,
         context=None,
+        check_pid=True,
```

Comment:
```suggestion
        _check_pid=True,
```
We don't want that to be exposed in the API.